### ISSUE TITLE
Overfitting Detection

### ISSUE THREAD

**Issue by akirs82** [ydwen/opensphere#22]
Hello Ydwen,

 I would like to know, from the training and validation logs how to check for overfitting or underfitting of the model capacity to the dataset ? My training loss is class_dataset type and my validation is pair_dataset type.

regards
akirs

**Comment by ydwen** [comment]
Hi akirs82,

You can construct a validation set of pair_dataset type, where the images are from *training set*.
This is the solution that we are currently using. Will have a push in the near future.

**Comment by akirs82** [comment]
Hello ydwen,
 Just to summarize.  I will split the dataset to training set (as class_dataset)  and validation set (in the form of pair_dataset type). Now i will have validation accuracy and training loss - with this should i get the graphs as below,
1. Validation accuracy vs Iteration
2. Training loss vs Iteration.
(Any pointers how these graphs will explain overfitting - as am seeing validation accuracy,training accuracy vs iteration or losses vs iteration)

I also have another dataset which is not part of the training set. Can i use this as well or not for validation set ?

 regards
 akirs

**Comment by MARUIHAN** [comment]
Hi, akir82. 
I am training my own dataset. how to construct the validation dataset with pair dataset type. Because my original dataset is a classification type dataset.
